src/merge.py

```
@@ -157,10 +138,13 @@ def validate_2fa(api: PyiCloudService) -> bool:
 
     return status
 
-def get_from_list(items:dict, value:str):
+def get_datetime(dt:datetime) -> datetime:
+    return datetime(dt.year, dt.month, dt.day, dt.hour, dt.minute, tzinfo=dt.tzinfo)
+
+def get_from_list(items:list, value:str):
     try:
         return_value = items.get(value)
-    except Exception:
+    except:
```

Ruff (0.15.6)

[error] Do not use bare `except`

(E722)